kitsune/sumo/management/commands/merge.py

```diff
@@ -31,42 +23,44 @@ def handle(self, *args, **options):
             if locale in ("en_US", "xx"):
                 continue
 
-            if Path(f"locale/{locale}").is_dir():
-                sub_cmd = "update"
-            elif not is_supported_for_init(locale):
-                # NOTE: Babel only supports initializing locales included in the CLDR.
-                self.stdout.write(
-                    self.style.WARNING(f"WARNING: skipping locale {locale} (unsupported for init)")
+            for domain in ("django", "djangojs"):
+
+                domain_pot = Path(f"locale/templates/LC_MESSAGES/{domain}.pot")
+                if not domain_pot.is_file():
+                    raise CommandError(f"unable to find {domain_pot}")
+                domain_po = Path(f"locale/{locale}/LC_MESSAGES/{domain}.po")
+                if not domain_po.parent.is_dir():
+                    domain_po.parent.mkdir(parents=True)
+
+                if not domain_po.is_file():
+                    self.run(
+                        "msginit",
+                        "--no-translator",
+                        f"--locale={locale}",
+                        f"--input={domain_pot}",
+                        f"--output-file={domain_po}",
+                        "--width=200",
+                        on_success=f"initialized {domain_po}",
+                        on_failure=f"error initializing {domain_po}",
+                    )
+
+                self.run(
```

Comment:
Do we need a context manager here?

Reply:
It would be also nice to have some kind of output on what's happening and in which locale